lightning-rapid-gossip-sync/src/lib.rs

```diff
@@ -10,24 +10,36 @@
 #![deny(unused_mut)]
 #![deny(unused_variables)]
 #![deny(unused_imports)]
-//! This crate exposes functionality to rapidly sync gossip data, aimed primarily at mobile
+//! This crate exposes client functionality to rapidly sync gossip data, aimed primarily at mobile
 //! devices.
 //!
-//! The server sends a compressed response containing differential gossip data. The gossip data is
-//! formatted compactly, omitting signatures and opportunistically incremental where previous
-//! channel updates are known (a mechanism that is enabled when the timestamp of the last known
-//! channel update is communicated). A reference server implementation can be found
-//! [here](https://github.com/lightningdevkit/rapid-gossip-sync-server).
+//! The rapid gossip sync server will provide a compressed response containing differential gossip
+//! data. The gossip data is formatted compactly, omitting signatures and opportunistically
+//! incremental where previous channel updates are known. This mechanism is enabled when the
+//! timestamp of the last known channel update is communicated. A reference server implementation
+//! can be found [on Github](https://github.com/lightningdevkit/rapid-gossip-sync-server).
 //!
-//! An example server request could look as simple as the following. Note that the first ever rapid
-//! sync should use `0` for `last_sync_timestamp`:
+//! The primary benefit of this syncing mechanism is that it allows a low-powered client to offload
+//! the validation of gossip signatures to a trusted server. This enables the client to privately
+//! calculate routes for payments, and to do so much faster than requiring a full peer-to-peer
+//! gossip sync to complete.
+//!
+//! The server calculates its repsonse on the basis of a client-provided `latest_seen` timestamp,
+//! i.e., it will return all rapid gossip sync data it has seen after the given timestamp.
```

Comment:
this phrasing is a bit more ambiguous than it was before, what with the timestamp being client-provided, and the snippet "data it has seen." It's not as clear that there, "it" is referring to the server.